viewing_party/party.py

```diff
@@ -1,23 +1,118 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
 
+    dict_movie={
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    for key in dict_movie:
+        if not dict_movie[key]:
+            return None
+    return dict_movie
+def add_to_watched(the_user_data, movie_watched):
+    the_user_data["watched"].append(movie_watched)
+    return the_user_data
+
+def add_to_watchlist(the_user_d, movie_to_watch):
+    the_user_d["watchlist"].append(movie_to_watch)
+    return the_user_d
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+            if movie["title"] == title:
+                user_data["watched"].append(movie)
+                i  = user_data["watchlist"].index(movie)

```

Comment:
This works totally fine, but it's worth mentioning to use caution when modifying a list that's currently being iterated over. If we were to test this in a scenario where `user_data["watchlist"]` has 2 elements, and the first is the movie that we are moving to `user_data["watched"]`, the second iteration through the loop will throw an index error, because it will try to access the 2nd element in `user_data["watchlist"]`, but there no longer is a second element. As a coding pattern this is one that I recommend avoiding, because it has so much potential for unforeseen errors. One fix is to return immediately after making the swap using the `break` keyword.